tlslite/utils/cryptomath.py

```diff
@@ -126,6 +126,12 @@ def bytesToNumber(b, endian="big"):
 
     By default assumes big-endian encoding of the number.
     """
+    # if string is empty, consider it to be representation of zero
+    # while it may be a bit unorthodox, it is the inverse of numberToByteArray
+    # with default parameters
```

Comment:
In Python 3.5.3 calling
```
int.from_bytes(b'', byteorder='big', signed=False)
```
will return 0 so it seems to be generally useful and expected from Python - only problem is that this is not available in 2.6 or 2.7, only 3.2 and later